runtime/onert/backend/train/MemoryPlanner.test.cc

```diff
@@ -15,19 +15,88 @@
  */
 
 #include <gtest/gtest.h>
+#include <stdexcept>
 
 #include "DisposableTensorIndex.h"
+#include "LayerScopeTensorIndex.h"
 #include "MemoryPlanner.h"
 #include "ir/Index.h"
 
 using namespace onert::backend::train;
 using onert::ir::OperandIndex;
 using onert::ir::OperationIndex;
 
-// TODO: Add test testcase for {Bump, FirstFit, WIC}Planner<LayerScopeTensor>
+namespace
+{
+
+template <typename T> T to_index(uint32_t first, uint32_t second)
+{
+  throw std::runtime_error{"unimplemented to_index"};
```

Comment:
How about using `static_assert` to check invalid type in compile time?
```suggestion
  static_assert{false, "unimplemented to_index"};
```

Reply:
I tried, but `static_assert(false)` is not allowed in compile time.
It looks like if `false` is in `static_assert` compiler figured out invalid-program and it fails to compile :cry: 

```
/home/ONE_TWO/runtime/onert/backend/train/MemoryPlanner.test.cc: In function ‘T {anonymous}::to_index(uint32_t, uint32_t)’:
/home/ONE_TWO/runtime/onert/backend/train/MemoryPlanner.test.cc:34:17: error: static assertion failed: unimplemented to_index<T>()
   34 |   static_assert(false, "unimplemented to_index<T>()");
      |                 
```

Reply:
Sorry for my wrong information :cry: 
Accroding to https://www.open-std.org/jtc1/sc22/wg21/docs/cwg_defects.html#941, you may be possible to apply deleted function templates to do it.
```cpp
template<typename T>
T to_index(uint32_t first, uint32_t second) = delete;
```

Reply:
Wow, I didn't know that `=delete;` can be used for this case. Thank you for searching :+1: 

Updated